CI fail on :

Checking formatting...
[warn] tests/unit/uri-template.test.ts
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
Error: Process completed with exit code 1.

I retry to check...

--- a/tools/ui/tests/unit/uri-template.test.ts
+++ b/tools/ui/tests/unit/uri-template.test.ts
@@ -107,11 +107,13 @@ describe('expandTemplate', () => {
 		expect(result).toBe('http://example.com?q=search%20term');
 	});
 
-    it('expands multiple query parameters', () => {
-        const result = expandTemplate('http://example.com{?q,sort}', { q: 'search term', sort: 'descending' });
-        expect(result).toBe('http://example.com?q=search%20term&sort=descending');
-    });
-
+	it('expands multiple query parameters', () => {
+		const result = expandTemplate('http://example.com{?q,sort}', {
+			q: 'search term',
+			sort: 'descending'
+		});
+		expect(result).toBe('http://example.com?q=search%20term&sort=descending');
+	});
 
 	it('keeps static parts unchanged', () => {
 		const result = expandTemplate('http://example.com/static', {});

